src/webgpu/gpu_test.ts

```diff
@@ -314,22 +314,12 @@ got [${failedByteActualValues.join(', ')}]`;
     this.expectContents(buffer, new Uint8Array(arrayBuffer));
   }
 
-  // TODO: Add check for values of depth/stencil, probably through sampling of shader
-  // TODO(natashalee): Can refactor this and expectSingleColor to use a similar base expect
-  expectSinglePixelIn2DTexture(
+  makeBufferFromTextureSinglePixel(
```

Comment:
oops, Shrek made the same refactor in #437. I'll rebase this on that when it lands.